nodeup/pkg/model/secrets.go

```diff
@@ -116,6 +116,44 @@ func (b *SecretBuilder) Build(c *fi.ModelBuilderContext) error {
 		c.AddTask(t)
 	}
 
+	if b.Cluster.Spec.KubeAPIServer.ProxyClientKeyFile != nil {
```

Comment:
Whoops. Yes, thanks for catching this. FWIW, the validation should catch if only one is specified.